### PR TITLE
fix: Get LimitChecker running from launcher. Updated LimitChecker package

### PR DESCRIPTION
**Pull request by PiRXlv**
(ARTP-858, ARTP-341)

### CODE REVIEW

File: src/client/src/apps/SimpleLauncher/tools.ts [hunk 1]
```diff
@@ -71,12 +71,20 @@ export async function open(
 }
 
 async function launchLimitChecker(config: ApplicationConfig) {
-  const app = fin.Application.wrap({ uuid: config.name })
-  fin.desktop.System.launchExternalProcess({
-    alias: 'LimitChecker',
-    listener(result) {
-      console.log('the exit code', result.exitCode)
+  const app = fin.Application.wrap({uuid: config.name})
+  console.log ("app", app);
```
Comment (sizov) [inline src/client/src/apps/SimpleLauncher/tools.ts]: what is this for? can we remove this please or make this log line more meaningful?

File: src/client/public/config/openfin/launcher-dev.json [hunk 2]
```diff
@@ -1,80 +1,85 @@
...
+            "reloadIgnoringCache": true,
+            "zoom": true
         }
-      },
-      "manifestUrl": "https://cdn.openfin.co/services/openfin/layouts/1.1.0/app.json"
     },
-    {
-      "name": "fdc3",
-      "manifestUrl": "https://cdn.openfin.co/services/openfin/fdc3/0.2.0/app.json"
+    "runtime": {
```
Comment (sizov) [inline src/client/public/config/openfin/launcher-dev.json]: am I right that you only added permissions as we discussed on the call?

Comment (PiRXlv) [reply src/client/public/config/openfin/launcher-dev.json]: yup

Comment (sizov) [reply src/client/public/config/openfin/launcher-dev.json]: thanks, the diff is confusing

**Pull request merged**